src/components/learner-credit-management/data/hooks/useCancelContentAssignments.js

```diff
@@ -0,0 +1,55 @@
+import { useCallback, useState } from 'react';
+import { logError } from '@edx/frontend-platform/logging';
+import { useToggle } from '@edx/paragon';
+import EnterpriseAccessApiService from '../../../../data/services/EnterpriseAccessApiService';
+
+const generateSuccessCancelMessage = (assignmentUuids) => {
+  if (Array.isArray(assignmentUuids)) {
+    return `Assignments canceled (${assignmentUuids.length})`;
+  }
+
+  return 'Assignment canceled';
+};
```

Comment:
[suggestion] I might recommend keeping the `assignmentUuids` function argument be consistent in its data type so it doesn't need to handle when its an array of UUIDs vs. a single UUID. The variable name itself implies multiple UUIDs as a list.